### DISCUSSION TITLE
Can't import TypeAliasType

### DISCUSSION THREAD

**Discussion by adriangb** [pydantic/pydantic#6621] [Question] [answered]
Moving https://github.com/python/typing_extensions/issues/159#issuecomment-1631451146 cc @gamaleonardo

I'm 100% certain the issue is with your dependencies and not Pydantic or typing-extensions. Please check the versions of your dependencies, e.g. `python -c 'import pydantic;print(pydantic.__version__)'`

**Comment by leonartdoss** [comment, marked as answer]
Yes @adriangb. I found the issue. In our application we use the layer AWSSDKPandas-Python39:7 which depends on typing_extensions ^4.4.0 (https://github.com/aws/aws-sdk-pandas/blob/main/pyproject.toml#L36). So there was a conflict between these two versions (4.5.0 from SDK Pandas and 4.7.1 from pydantic). I upgraded it to version 8 (AWSSDKPandas-Python39:8) and it solved the issue.

Thanks for your feedback
